modules/cli/src/main/scala/scala/cli/commands/package0/PackagerOptions.scala

```diff
@@ -132,6 +132,12 @@ final case class PackagerOptions(
   )
   @Tag(tags.restricted)
   dockerImageTag: Option[String] = None,
+  @Group(HelpGroup.Docker.toString)
+  @HelpMessage(
+    "Executable that will run an application in docker, default sh"
```

Comment:
technically, the platform influences what it defaults to, no?
```suggestion
    "Allows to override the executable used to run the application in docker, otherwise it defaults to sh for the JVM platform and node for the JS platform"
```